lib/timezone/lookup/geonames.rb

```diff
@@ -25,9 +25,7 @@ def lookup(lat, long)
 
         data = JSON.parse(response.body)
 
-        return data['timezoneId'] if data['timezoneId']
-
-        return unless data['status']
+        return get_timezone_id(data) unless data['status']
```

Comment:
One final comment. This is a bit of a behavior change. I don't want to end up in the case where a `timezoneId` and `status` is returned and we fail to return.

```
timezoneId: 'America/Los_Angeles',
stauts: 'success'
```

Can we just inline some of the functionality to revert this?

```
return data['timezoneId'] if data['timezoneId']

if data['gmtOffset'] && config.offset_etc_zones
  return calculate_gmt_offset(data['gmtOffset'])
end

return unless data['status']

...
```